docs/source/securing-calico.rst

```diff
@@ -78,9 +78,99 @@ traffic that should be allowed.
 Securing etcd
 -------------
 
+Limiting network access to etcd
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 Calico uses etcd to store and forward the configuration of the network from
 plugin to the Felix agent.  By default, etcd is writable by anyone with
 access to its REST interface.  We plan to use the RBAC feature of an upcoming
 etcd release to improve this dramatically.  However, until that work is done,
 we recommend blocking access to etcd from all but the IP range(s) used by the
 compute nodes and plugin.
+
+.. _usingtlswithetcd:
+
+Using TLS to encrypt and authenticate communication with etcd
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Calico supports etcd's TLS-based security model, which supports the encryption
+(and authentication) of traffic between Calico components and the etcd cluster.
+
+.. warning:: Calico's TLS support uses the Python urllib3 library.
+             Unfortunately, we've seen issues with TLS in some of the common
+             versions of urllib3 including the version that ships with
+             Ubuntu 14.04 (1.7.1) and versions 1.11-1.12.  Version 1.13
+             appears to work well.
```

Comment:
Is it useful to splash this warning if we don't say anything about what those issues are?